crates/cxx-qt-gen/src/generator/rust/qobject.rs

```diff
@@ -33,6 +33,7 @@ impl GeneratedRustQObjectBlocks {
     }
 }
 
+#[derive(Debug)]
```

Comment:
any reasoning for these extra derive Debug's ? They could be done in a separate commit if they are useful ...

Reply:
Mostly for debugging, I'll remove then